.github/workflows/chromatic.ds-react-core-form.yml

```
@@ -0,0 +1,11 @@
+name: "react-ds-core/chromatic"
+on:
+  pull_request:
+    paths: [packages/react/ds-core/**]
```

Core and form have been mixed up
```suggestion
    paths: [packages/react/ds-core-form/**]
```